mephisto/client/review/review_server.py

```diff
@@ -41,16 +42,39 @@ def json_reader(f):
         for jsonline in f:
             yield json.loads(jsonline)
 
+    def mephistoDBReader():
+        if database is None:
+            return []
+        from mephisto.abstractions.databases.local_database import LocalMephistoDB
+        from mephisto.tools.data_browser import DataBrowser as MephistoDataBrowser
+        from mephisto.data_model.worker import Worker
+        from mephisto.data_model.unit import Unit
+
+        db = LocalMephistoDB()
+        mephisto_data_browser = MephistoDataBrowser(db=db)
+        units = mephisto_data_browser.get_units_for_task_name(database)
+
+        def format_data_for_review(data):
+            contents = data["data"]
+
+            inputs = contents["inputs"]
+            inputs_string = f"Character: {inputs['character_name']}, Description: {inputs['character_description']}"
```

Comment:
Where are `'character_name'` and `'character_description'` coming from here? Is this specific nto the `static_example` task? Not all tasks will have this data schema, we should generalize